set.mm

```diff
@@ -538886,6 +538891,29 @@ is a Moore collection ( ~ bj-snmoore ), this can also be stated as
       VAUTMFDRWDVBVCVDVEVF $.
   $}
 
+  ${
+    $d A x $.
+    $( A pair formed of the empty set and any class is a Moore collection (on
+       ` U. { (/) , A } ` , which is ` A ` if it is a set, else ` (/) ` ).
+
+       One also has ` |- ( ( A C_ B /\ A e. V ) -> { A , B } e. Moore_ ) ` ,
```

Comment:
Do you plan to provide this as theorem soon?

Reply:
Yes, it's only a small variant of that theorem (basically, replace in the proof the line $\varnothing \cap A= \varnothing$ by $A \subseteq B \to A \cap B = A$), but I won't have time in this MR.

Reply:
Wouldn't it be easier to prove the empty set version from the subset version?
I.e. to prove that coming theorem from this version?

Reply:
Of course: once I prove the general theorem, the one with the empty set will be a direct consequence (as will be the forward implication in `|- ( A e. _V <-> { A } e. Moore_ )`).